schemas/tsp/customer-auth-validate/response.json

```diff
@@ -17,7 +17,7 @@
     }
   },
   "additionalProperties": false,
-  "oneOf": [
+  "anyOf": [
```

Comment:
I think the problem here is that `["authToken", "nonce", "validTo", "error"]` option makes no sense.. as payloads that satisfy it, will also always satisfy `["authToken", "nonce", "validTo"]`.

Therefore good fix would be to remove the middle option and keep it as `oneOf`

Reply:
Yes, that's the issue. But then again removing middle option and keeping oneOf fails too.

Reply:
> removing middle option and keeping oneOf fails too.

I think in that case it may only fail for invalid request. Can you provide an example of payload for which it fails, why you feel it shouldn't?

Reply:
```json
{"authToken":"xxx","validTo":1582720901449,"nonce":"xxx","error":{"code":"NOT_ELIGIBLE","message":"INELIGIBLE_USER"}}
```

Which is what we also need, the middle one.

Reply:
Ok, got it. Issue is that response having `authToken`, `nonce`, `validTo` and `error`.. will also match ` ["error", "nonce"]` version, hence the crash with `oneOf` then.

Indeed we should change it to `anyOf` then. However middle option still should be removed as it's just rudimentary (doesn't extend validation with new possible options)

Reply:
Thanks for help